homeassistant/components/automation/__init__.py

```diff
@@ -249,6 +242,14 @@ def __init__(self, unique_id, config, trigger, context, action_trace):
         self._unique_id = unique_id
         self._variables = None
 
+    def set_action_trace(self, condition_trace):
```

Comment:
Can you add typing to this function and class.

Reply:
Fixed in b1c4e2c